test/io/ably/test/rest/RestInit.java

```diff
@@ -11,11 +7,12 @@
 import io.ably.types.ClientOptions;
 import io.ably.util.Log;
 import io.ably.util.Log.LogHandler;
+import org.junit.Test;
 
 import java.io.ByteArrayOutputStream;
 import java.io.PrintStream;
 
-import org.junit.Test;
+import static org.junit.Assert.*;
```

Comment:
Please use the same style for imports as the existing code (ie without wildcards).